[ 1379s] I: Program returns random data in a function
[ 1379s] E: nodejs42 no-return-in-nonvoid-function ../deps/v8/src/builtins/builtins-intl.cc:749
[ 1379s] E: nodejs42 no-return-in-nonvoid-function ../deps/v8/src/compiler/effect-control-linearizer.cc:4122
[ 1379s] E: nodejs42 no-return-in-nonvoid-function ../deps/v8/src/compiler/js-heap-broker.cc:2277
[ 1379s] E: nodejs42 no-return-in-nonvoid-function ../deps/v8/src/objects/js-break-iterator.cc:167
[ 1379s] E: nodejs42 no-return-in-nonvoid-function ../deps/v8/src/objects/js-date-time-format.cc:155
[ 1379s] E: nodejs42 no-return-in-nonvoid-function ../deps/v8/src/objects/js-list-format.cc:86, 231, 244
[ 1379s] E: nodejs42 no-return-in-nonvoid-function ../deps/v8/src/objects/js-locale.cc:375, 390, 403
[ 1379s] E: nodejs42 no-return-in-nonvoid-function ../deps/v8/src/objects/js-number-format.cc:472, 485
[ 1379s] E: nodejs42 no-return-in-nonvoid-function ../deps/v8/src/objects/js-relative-time-format.cc:40, 193, 204
[ 1379s] E: nodejs42 no-return-in-nonvoid-function ../deps/v8/src/objects/js-segmenter.cc:196, 211
[ 1379s]

[ 4759s] E: nodejs12 no-return-in-nonvoid-function ../deps/v8/src/compiler/code-assembler.h:284
[ 4759s] E: nodejs12 no-return-in-nonvoid-function ../deps/v8/src/objects/js-collator.cc:229
[ 4759s] E: nodejs12 no-return-in-nonvoid-function ../deps/v8/src/objects/js-date-time-format.cc:344, 587, 989
[ 4759s] E: nodejs12 no-return-in-nonvoid-function ../deps/v8/src/objects/js-number-format.cc:40
[ 4759s] E: nodejs12 no-return-in-nonvoid-function ../deps/v8/src/objects/js-plural-rules.cc:92
[ 4759s] E: nodejs12 no-return-in-nonvoid-function ../deps/v8/src/objects/js-segment-iterator.cc:43, 125
[ 4759s] E: nodejs12 no-return-in-nonvoid-function ../deps/v8/src/vm-state-inl.h:40
[ 4759s] 

--- a/deps/v8/src/compiler/effect-control-linearizer.cc
+++ b/deps/v8/src/compiler/effect-control-linearizer.cc
@@ -3821,6 +3821,8 @@ Node* EffectControlLinearizer::BuildReve
     case kExternalBigUint64Array:
       UNREACHABLE();
   }
+
+  UNREACHABLE();
 }
 
 Node* EffectControlLinearizer::LowerLoadDataViewElement(Node* node) {
--- a/deps/v8/src/compiler/js-heap-broker.cc
+++ b/deps/v8/src/compiler/js-heap-broker.cc
@@ -1090,6 +1090,8 @@ MapRef NativeContextRef::GetInitialJSArr
     default:
       UNREACHABLE();
   }
+
+  UNREACHABLE();
 }
 
 bool ObjectRef::BooleanValue() {
--- a/deps/v8/src/objects/js-list-format.cc
+++ b/deps/v8/src/objects/js-list-format.cc
@@ -83,6 +83,8 @@ const char* GetIcuStyleString(JSListForm
     case JSListFormat::Type::COUNT:
       UNREACHABLE();
   }
+
+  UNREACHABLE();
 }
 
 }  // namespace
@@ -233,6 +235,8 @@ Handle<String> JSListFormat::StyleAsStri
     case Style::COUNT:
       UNREACHABLE();
   }
+
+  UNREACHABLE();
 }
 
 Handle<String> JSListFormat::TypeAsString() const {
@@ -246,6 +250,8 @@ Handle<String> JSListFormat::TypeAsStrin
     case Type::COUNT:
       UNREACHABLE();
   }
+
+  UNREACHABLE();
 }
 
 namespace {
--- a/deps/v8/src/objects/js-relative-time-format.cc
+++ b/deps/v8/src/objects/js-relative-time-format.cc
@@ -37,6 +37,8 @@ UDateRelativeDateTimeFormatterStyle getI
     case JSRelativeTimeFormat::Style::COUNT:
       UNREACHABLE();
   }
+
+  UNREACHABLE();
 }
 }  // namespace
 
@@ -197,6 +199,8 @@ Handle<String> JSRelativeTimeFormat::Sty
     case Style::COUNT:
       UNREACHABLE();
   }
+
+  UNREACHABLE();
 }
 
 Handle<String> JSRelativeTimeFormat::NumericAsString() const {
@@ -208,6 +212,8 @@ Handle<String> JSRelativeTimeFormat::Num
     case Numeric::COUNT:
       UNREACHABLE();
   }
+
+  UNREACHABLE();
 }
 
 }  // namespace internal
--- a/deps/v8/src/objects/js-collator.cc
+++ b/deps/v8/src/objects/js-collator.cc
@@ -535,6 +535,8 @@ const char* JSCollator::UsageToString(Us
     case Usage::COUNT:
       UNREACHABLE();
   }
+
+  return UCOL_DEFAULT;
 }
 
 }  // namespace internal
--- a/deps/v8/src/vm-state-inl.h
+++ b/deps/v8/src/vm-state-inl.h
@@ -37,6 +37,7 @@ inline const char* StateToString(StateTa
     default:
       UNREACHABLE();
   }
+  return "SHOULD_NOT_HAPPEN";
 }
 
 
